packages/sdk/src/constants/gas-controllers/moe-collection-deploy.ts

```diff
@@ -139,4 +139,16 @@ export const MOE_COLLECTION_DEPLOY_GAS = {
     gasLimitMultiplier: 120,
     gasPriceMultiplier: 110,
   },
+  [networks.localhost.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 120,
+    gasPriceMultiplier: 110,
+  },
+  [networks.localhost2.chain]: {
+    gasPrice: undefined,
+    gasLimit: undefined,
+    gasLimitMultiplier: 120,
```

Comment:
Are these values essentially overhead multipliers? in other words, is this like 120%?

Reply:
yep, i think so

copied from the FE implementation